--- xbmc/configure.in	2012-10-21 22:41:20.368057835 +0200
+++ xbmc-build/configure.in	2012-10-21 22:53:53.511585402 +0200
@@ -1150,6 +1150,15 @@
         AC_MSG_RESULT($pulse_not_found)
       fi])
       USE_PULSE=1
+    AC_CHECK_LIB([pulse-simple], [main],,
+      [if test "x$use_pulse" = "xyes"; then
+        AC_MSG_ERROR($pulse_not_found)
+      else
+        use_pulse=no
+        USE_PULSE=0
+        AC_MSG_RESULT($pulse_not_found)
+      fi])
+      USE_PULSE=1
   fi
 else
   AC_MSG_RESULT($pulse_disabled)
