diff -uNr cxfe-0.9.2/cxfe.c cxfe-0.9.2-patched/cxfe.c
--- cxfe-0.9.2/cxfe.c	2010-08-24 07:23:56.000000000 +0200
+++ cxfe-0.9.2-patched/cxfe.c	2010-08-24 19:40:22.000000000 +0200
@@ -1286,6 +1286,23 @@
      window[1] = XCreateSimpleWindow(display, XDefaultRootWindow(display),
 				     0, 0, res_v, res_h, 0, 0, 0);
 
+     {
+        Cursor invisibleCursor;
+        Pixmap bitmapNoData;
+        XColor black;
+        static char noData[] = { 0,0,0,0,0,0,0,0 };
+
+        black.red = black.green = black.blue = 0;
+        bitmapNoData = XCreateBitmapFromData(display, window[0], noData, 8, 8);
+        invisibleCursor = XCreatePixmapCursor(display, bitmapNoData, bitmapNoData, &black, &black, 0, 0);
+        XDefineCursor(display, window[0], invisibleCursor);
+        XFreeCursor(display, invisibleCursor);
+        bitmapNoData = XCreateBitmapFromData(display, window[1], noData, 8, 8);
+        invisibleCursor = XCreatePixmapCursor(display, bitmapNoData, bitmapNoData, &black, &black, 0, 0);
+        XDefineCursor(display, window[1], invisibleCursor);
+        XFreeCursor(display, invisibleCursor);
+     }
+
      if (fullscreen) {
        width = res_v;
        height = res_h;
